### DISCUSSION TITLE
Possibilty of Apple Signin?

### DISCUSSION THREAD

**Discussion by Hannes1** [supabase/supabase#1339] [Questions] [answered]
Hi, I'm wondering if there will be support for apple authentication in the future. I'm building react native application but for an app to be accepted on the apple app store you need apple auth if you want to use google sign in sigh...

**Comment by nicogenz** [comment, marked as answer]
There is already an open issue, which tracks this requirement. See [Apple Login](https://github.com/supabase/gotrue/issues/37)

**Reply by inian** [reply]
This has been implemented now
